UnitTests/BraintreePaymentFlowTests/BTLocalPaymentRequest_Tests.swift

```diff
@@ -33,39 +29,20 @@ class BTLocalPaymentRequest_UnitTests: XCTestCase {
         request.isShippingAddressRequired = true
         request.displayName = "My Brand!"
 
-        let mockRequestDelegate = MockLocalPaymentRequestDelegate()
-        request.localPaymentFlowDelegate = mockRequestDelegate
-
-        let mockClientDelegate = MockPaymentFlowClientDelegate()
-        mockClientDelegate._returnURLScheme = "com.app.payments"
-
-        request.handle(request, client: mockClient, paymentClientDelegate: mockClientDelegate)
-
-        XCTAssertEqual(mockClient.lastPOSTPath, "v1/local_payments/create")
-
-        guard let params = mockClient.lastPOSTParameters else { XCTFail(); return }
-
-        XCTAssertEqual(params["amount"] as! String, "100.00")
-        XCTAssertEqual(params["funding_source"] as! String, "payment-type")
-        XCTAssertEqual(params["intent"] as! String, "sale")
-        XCTAssertEqual(params["return_url"] as! String, "sdk.ios.braintree://x-callback-url/braintree/local-payment/success")
-        XCTAssertEqual(params["cancel_url"] as! String, "sdk.ios.braintree://x-callback-url/braintree/local-payment/cancel")
-        XCTAssertEqual(params["payment_type_country_code"] as! String, "US")
-        XCTAssertEqual(params["line1"] as! String, "street-address")
-        XCTAssertEqual(params["line2"] as! String, "extended-address")
-        XCTAssertEqual(params["city"] as! String, "Chicago")
-        XCTAssertEqual(params["state"] as! String, "IL")
-        XCTAssertEqual(params["postal_code"] as! String, "12345")
-        XCTAssertEqual(params["country_code"] as! String, "US")
-        XCTAssertEqual(params["first_name"] as! String, "Jane")
-        XCTAssertEqual(params["last_name"] as! String, "Doe")
-        XCTAssertEqual(params["payer_email"] as! String, "test@example.com")
-        XCTAssertEqual(params["phone"] as! String, "1231231234")
-        XCTAssertEqual(params["merchant_account_id"] as! String, "account-id")
-        XCTAssertEqual(params["bic"] as! String, "bank-id-code")
-        
-        let experienceProfile = params["experience_profile"] as! [String: Any]
-        XCTAssertEqual(experienceProfile["brand_name"] as! String, "My Brand!")
-        XCTAssertEqual(experienceProfile["no_shipping"] as! Bool, false)
+        XCTAssertEqual(request.amount, "100.00")
```

Comment:
Previously this tested that the POST params to `v1/local_payments/create` were properly formatted and translated from the merchant's `BTLocalPaymentRequest`. Now this test looks to be just asserting the params set on the `request`.

I think we still want to preserve this test, it will likely just have to move since the request isn't doing the post anymore

Reply:
This test already existed in basically the same format here: https://github.com/braintree/braintree_ios/blob/2bbbf1147ba25901c59522dd68cac09c81fe97c2/UnitTests/BraintreePaymentFlowTests/BTLocalPayment_Tests.swift#L101 Though I did just notice it's missing a few values so I've added those in this commit: https://github.com/braintree/braintree_ios/pull/1035/commits/c78fca5da46154f176158bf141cd7d527e33418d

The request test is now updated to match the format of our other request test files: [ThreeDSecureRequest_Tests](https://github.com/braintree/braintree_ios/blob/2bbbf1147ba25901c59522dd68cac09c81fe97c2/UnitTests/BraintreeThreeDSecureTests/BTThreeDSecureRequest_Tests.swift)

Reply:
Got it. I didn't see that this test basically lives now at `BTLocalPayment_UnitTests` --> `testStartPayment_postsAllCreationParameters()`. 

We should rename `BTLocalPayment_UnitTests` --> `BTLocalPaymentClient_UnitTests`. The SUT for all of the tests in that file is `LocalPaymentClient`.

Reply:
> We should rename BTLocalPayment_UnitTests --> BTLocalPaymentClient_UnitTests. The SUT for all of the tests in that file is LocalPaymentClient.

Done: https://github.com/braintree/braintree_ios/pull/1035/commits/eb8925e7f9fb832a8be85a14a7254c54369927b9

Reply:
> The remaining test in BTLocalPaymentRequest_Tests doesn't provide value anymore. I think we can 🔥 it.

Always down to 🔥  done: https://github.com/braintree/braintree_ios/pull/1035/commits/5c23a4798f6c09e65f98ac669ee4dcfc1ce1b64c